docs/source/blogs/Best_perf_practice_on_DSR1_in_TRT-LLM.md

```diff
@@ -0,0 +1,362 @@
+Average request latency (ms):                     174119.3317
+```
+
+## H200 NVL8
+### Prerequisites
+
+``` bash
+# Prerequisites
+apt-get update && apt-get -y install git git-lfs
+git lfs install
+
+# Replace with your actual path
+YOUR_WORK_PATH=<YOUR_WORK_PATH>
+YOUR_MODEL_PATH=<YOUR_MODEL_PATH>
+YOUR_DATA_PATH=<YOUR_DATA_PATH>
+
+# Clone the TensorRT-LLM repository
+cd $YOUR_WORK_PATH
+git clone https://github.com/NVIDIA/TensorRT-LLM.git
+cd TensorRT-LLM
+git submodule update --init --recursive
+git lfs pull
+
+# Clone the DeepSeek-R1 model
+cd $YOUR_MODEL_PATH
+GIT_LFS_SKIP_SMUDGE=1 git clone https://huggingface.co/deepseek-ai/DeepSeek-R1
+git lfs pull  # Download the full model weight will take a long time
+```
+**Note**: Replace `<*_PATH>` to your actual path. 
+
+#### Build Docker 
+Create a docker and run:
+
+``` bash
+cd TensorRT-LLM
+make -C docker jenkins_run LOCAL_USER=1 DOCKER_RUN_ARGS="-v $YOUR_MODEL_PATH:$YOUR_MODEL_PATH:ro -v $YOUR_DATA_PATH:$YOUR_DATA_PATH:ro -v $YOUR_WORK_PATH:$YOUR_WORK_PATH"
+```
+Here we set `LOCAL_USER=1` argument to set up the local user account inside the container.
+
+#### Compile and Install
+Here we compile the source inside the container:
+
+``` bash
+python3 ./scripts/build_wheel.py --trt_root /usr/local/tensorrt --benchmarks --use_ccache --cuda_architectures "90-real"  --python_bindings --clean
+```
+
+Install and set environment variables:
+
+```bash
+pip install --user build/tensorrt_llm*.whl
+export PATH=${HOME}/.local/bin:${PATH}
+export PYTHONPATH=`pwd`
+```
+### H200 min-latency
+Our benchmark results are based on **Batch = 1, ISL = 1K, OSL = 2K, num_requests = 10 from real dataset**
+
+#### Benchmark
+To do the benchmark, run the following command:
+
+```bash
+# Enable DeepGEMM
+export TRTLLM_DG_ENABLED=1
+
+DS_R1_MODEL_PATH=$YOUR_MODEL_PATH/DeepSeek-R1
+trtllm-bench --model deepseek-ai/DeepSeek-R1 \
+    --model_path $DS_R1_MODEL_PATH \
+    throughput \
+    --dataset $YOUR_DATA_PATH \
+    --backend pytorch \
+    --num_requests 10 \
+    --max_batch_size 1 \
+    --tp 8 \
+    --ep 4 \
+    --concurrency 1 \
+    --extra_llm_api_options ./extra-llm-api-config.yml
+```
+
+Explanation:
+- `trtllm-bench`: A CLI packags benchmarking utility that aims to make it easier for users to reproduce our officially published. [TensorRT-LLM Benchmarking](https://nvidia.github.io/TensorRT-LLM/performance/perf-benchmarking.html).
+- `--dataset`: Prompt dataset used to benchmark. our official benchmark dataset has ISL = 1K, OSL = 2K
+- `--backend`: Inference backend. Here we use Pytorch backed. 
+- `--tp 8`: Tensor parallel size is 8.
+- `--ep 4`: Expert parallel size is 4.
+- `--extra_llm_api_options`: Used to specify some extra config. The content of the file is as follows:
+
+    ``` yaml
+    pytorch_backend_config:
+        enable_overlap_scheduler: true
+        use_cuda_graph: true
+    speculative_config:
+        decoding_type: MTP
+        num_nextn_predict_layers: 3
+    ```
```

Comment:
These are exactly the same as B200. No need to duplicate the exact the instructions. We should refactor the instructions and only mention the difference on H200 when necessary.